.github/workflows/build_wheels.yml

```
@@ -87,9 +87,9 @@ jobs:
       - name: Build sdist
         run: python setup.py sdist
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4.0.0
```

```suggestion
      - uses: actions/upload-artifact@v4.1.0
```
4.1.0 was released moments ago. Save a future dependabot PR by using it now.